napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -78,6 +81,11 @@ def set_csv_name(self, name: Optional[str] = None):
         """Set csv file name for writing."""
         self.csv_name = name
 
+    def write_json(self, file_path: str):
+        """Write annotation dictionary to a file."""
+        if self.annot_json_data is not None:
+            json.dump(self.annot_json_data, open(file_path, "w"), indent=4)
```

Comment:
Test to see if this writes the javascript booleans correctly (it should be `true`). If not- use json.dumps.

Reply:
This works :)